FSharp-9.0/FS-1043-extension-members-for-operators-and-srtp-constraints.md

```diff
@@ -8,8 +8,8 @@ These design suggestions:
 have been marked "approved in principle". This RFC covers the detailed proposal for these
 
 * [x] Approved in principle
-* [Discussion thread](https://github.com/fsharp/fslang-design/issues/435)
-* [ ] Implementation: [In progress](https://github.com/dotnet/fsharp/pull/6805)
+* [x] [Discussion](https://github.com/fsharp/fslang-design/issues/435)
+* [x] [Implementation](https://github.com/dotnet/fsharp/pull/8404)
```

Comment:
This one isn't in F# 9

Reply:
Ah, I see my mixup. Was there an RFC for the "Prefer extension methods to intrinsic properties when arguments are provided" bit that was added in F# 9? https://learn.microsoft.com/en-us/dotnet/fsharp/whats-new/fsharp-9#prefer-extension-methods-to-intrinsic-properties-when-arguments-are-provided

Reply:
Moved FS-1043 back